src/ui/projects/api.py

```diff
@@ -449,3 +447,39 @@ def check_api(self) -> bool:
             return False
 
         return True
+
+    def get_objects(
+        self, project_id: int, job_id: int, start: int, length: int
+    ) -> Optional[Tuple[List[Object], int]]:
+        """Get a list of Objects for `job_id` from `start` to `start + lenght`.
+
+        Parameters
+        ----------
+        project_id : int
+            Project the job is part of.
+        job_id : int
+            Job the objects are part of.
+        start : int
+            First Object to be returned in response(0 index based).
+        lenght : int
+            Number of objects to be returned.
+
+        Returns
+        -------
+        Optional[Tuple[List[Object], int]]
+            List of objects, total number of Objects in job.
+        """
+        url = f"{self._endpoint}/projects/{project_id}/jobs/{job_id}/objects"
+        logger.info(f"Get objects from {url}")
+        response = self.get(
+            url,
+            params={"start": start, "length": length},
+        )
+
+        if isinstance(response, requests.Response):
+            dct = response.json()
+            objects = [Object(**d) for d in dct["data"]]
+            total_object_count = dct["total_objects"]
+            return objects, total_object_count
```

Comment:
Total amount of object should also be in `Job`/`JobBare` as part of `JobStat` if I remember correctly.

Reply:
The total amount is needed to be part of response to Datatable, so the object endpoint is returning it.

Reply:
I see!